engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkRestManager.scala

```diff
@@ -95,11 +96,19 @@ class FlinkRestManager(config: FlinkConfig, modelData: ModelData, mainClassName:
       case JobStatus.RECONCILING | JobStatus.CREATED | JobStatus.SUSPENDED => FlinkStateStatus.Running
       case JobStatus.FAILING => FlinkStateStatus.Failing
       case JobStatus.FAILED => FlinkStateStatus.Failed
-      case JobStatus.INITIALIZING => FlinkStateStatus.DuringDeploy
     }
 
   }
 
+  protected def ensureTasksRunning(overview: JobOverview): Boolean = {
+    overview.tasks.running + overview.tasks.finished == overview.tasks.total
+  }
+
+  protected def isDuringDeploy(s: JobStatus): Boolean = {
```

Comment:
canBeDuringDeploy? now it also returns true if job is running stably

Reply:
changed